Path-Check/gaen-mobile
# Add Eula and Privacy Policy screen

#### Description:

Why:
As a user, I want to be able to read and agree to my Health Authority's
Privacy Policy and EULA if they have them.

This commit:
Reintroduces the UI to accept the EULA as part of the activation flow,
and updates it according to the new design specs.

Note that we do not currently have a strategy for hosting a HA's EULA,
so actually shipping this work will be blocked on having a EULA to link
to externally.

#### Linked issues:

Fixes: [Terms of Use (EULA & Privacy Policy)](https://trello.com/c/MSEdWrmm/389-terms-of-use-eula-privacy-policy)

#### Screenshots:

![Simulator Screen Shot - iPhone 11 - 2020-08-25 at 10 40 02](https://user-images.githubusercontent.com/21161427/91189711-c4516780-e6c0-11ea-9610-e3b397eb0bee.png)
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-08-25 at 10 48 18](https://user-images.githubusercontent.com/21161427/91189712-c5829480-e6c0-11ea-83c5-f42a643dcf4b.png)


#### How to test:
- On a fresh install, navigate through the onboarding screen
- Expect to see the EULA and privacy policy before activating bluetooth or ENs
